src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -97,7 +106,8 @@ public void checkSourceContent(String desiredContent) {
   }
 
   public void checkSourceVideoContent(String desiredContent) {
-    Assertion.assertEquals(getSourceContent().substring(1, 38) + getSourceContent().substring(48), desiredContent.substring(1, 38) + desiredContent.substring(48)
+    Assertion.assertEquals(getSourceContent().substring(1, 38) + getSourceContent().substring(48),
```

Comment:
This change was caused probably by my styling scheme. I reverted it to previous state. I don't feel like I have enough context to describe it right now. So should I dig into it more deeply and prepare such a comment?